services/src/main/java/io/druid/cli/CliCoordinator.java

```diff
@@ -211,5 +218,16 @@ public LoadQueueTaskMaster getLoadQueueTaskMaster(
           }
         }
     );
+
+    if (beOverlord) {
+      modules.addAll(new CliOverlord().getModules(false));
+    }
+
+    return modules;
+  }
+
+  public static boolean isOverlord(Properties properties)
+  {
+    return !"false".equals(properties.getProperty("druid.coordinator.beOverlord"));
```

Comment:
Would prefer a tighter check here. As-is, setting to "no" or "FALSE" or "false " with some whitespace would make the coordinator be an overlord. How about throwing an exception if the property is set, but is not "true" or "false"?

Or, at the very least, log a warning.